frontend/packages/ux-editor/src/components/FormComponent/DragHandle.module.css

```diff
@@ -1,24 +0,0 @@
-.handle {
```

Comment:
Is this file unused? I see that files that are not related to debounce are deleted. I'm not sure why.  Any reason for it?😊